### PR TITLE
New package: LeawoSoftware.Blu-rayPlayer version 3.0.0.2

### LINKED ISSUE #92271: [Package Request]: Leawo Blu-ray Player
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not contained in a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Leawo
Publisher Url: https://www.leawo.com/blu-ray-player/
Package Name: Leawo Blu-ray Player
Description: Free blu-ray player with menu support
Package Version: latest version (currently 3.0.0.2)
Installer URL:
https://www.leawo.com/downloads/blu-ray-player.html
Current Package Identifier (for new versions):

### PR DESCRIPTION
**Pull request by mdanish-kh**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
- Closes microsoft/winget-pkgs#92271

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92282)

### CODE REVIEW

**Review by russellbanks — APPROVED** [review]

**Comment by stephengillie** [comment]
JRE and VCRedist (2010?) appear to be application dependencies.
image

- The JRE Update window pops up every time the application is launched.
- The VCRedist error is only encountered when running the updater. 

- Winget only installs the first dependency. 
  * https://github.com/microsoft/winget-cli/issues/2202

- JRE as dependency has same situation as Git as dependency: it's added to the Windows Path via Registry. This means it's in the Path for all future console windows, but isn't yet in the Path variable for the current windows. 
  * https://github.com/microsoft/winget-pkgs/issues/90650

File: manifests/l/LeawoSoftware/Blu-rayPlayer/3.0.0.2/LeawoSoftware.Blu-rayPlayer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: LeawoSoftware.Blu-rayPlayer
+PackageVersion: 3.0.0.2
+InstallerType: inno
+UpgradeBehavior: install
+ReleaseDate: 2022-12-12
+Installers:
```
Comment (stephengillie) [inline manifests/l/LeawoSoftware/Blu-rayPlayer/3.0.0.2/LeawoSoftware.Blu-rayPlayer.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
    - PackageIdentifier: Microsoft.VCRedist.2010.x64
Installers:
```

**Pull request merged**